jgrapht-core/src/main/java/org/jgrapht/generate/RandomRegularGraphGenerator.java

```diff
@@ -0,0 +1,276 @@
+/*
+ * (C) Copyright 2018, by Emilio Cruciani and Contributors.
```

Comment:
Please update the copyright header to have the following format

- (C) Copyright 2018-2018, by Emilio Cruciani and Contributors.

Headers are updated automatically; Deviation of the standard header messes this up.